casper/blockchain/blockchain_plot_tool.py

```diff
@@ -12,13 +12,16 @@ def __init__(self, display, save, view, validator_set):
         super().__init__(display, save, 's')
         self.view = view
         self.validator_set = validator_set
+        self.genesis_block = self.view.last_finalized_block
```

Comment:
Assumes the plot tool is created when the view is. Seems fine, but just noting.

Reply:
equal to `self.view.genesis_block` now